web3/process.go

```diff
@@ -119,90 +138,137 @@ func (c *Contracts) StateRoot(processID []byte) (*types.BigInt, error) {
 // the process. It returns the transaction hash of the state transition
 // submission, or an error if the submission fails. The tx hash can be used to
 // track the status of the transaction on the blockchain.
-func (c *Contracts) SetProcessTransition(processID, proof, inputs []byte, blobsSidecar *gtypes.BlobTxSidecar) (*common.Hash, error) {
+func (c *Contracts) SetProcessTransition(processID, proof, inputs []byte, blobsSidecar *gtypes.BlobTxSidecar) ([]byte, *common.Hash, error) {
+	// If the transaction manager is not available, return an error
+	if c.txManager == nil {
```

Comment:
We must assume that txManager is always initialized. So, I won't do this check here (nor in other functions). If txManager is not initialized, just let the code Panic.